docs/modules/ROOT/examples/management-center-node-affinity.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: ManagementCenter
+metadata:
+  name: managementcenter
+spec:
+  scheduling:
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: kubernetes.io/arch
```

Comment:
Trivial but indentation for `expression` list in MC file is different from Hazelcast.